packages/editor/src/browser/editor-manager.ts

```diff
@@ -54,13 +54,16 @@ export class EditorManager extends NavigatableWidgetOpenHandler<EditorWidget> {
      */
     readonly onCurrentEditorChanged: Event<EditorWidget | undefined> = this.onCurrentEditorChangedEmitter.event;
 
+    @inject(CommandService) protected readonly commands: CommandService;
     @inject(PreferenceService) protected readonly preferenceService: PreferenceService;
 
     @postConstruct()
     protected override init(): void {
         super.init();
         this.shell.onDidChangeActiveWidget(() => this.updateActiveEditor());
-        this.shell.onDidChangeCurrentWidget(() => this.updateCurrentEditor());
```

Comment:
I think this was accidentally deleted? We should probably keep this line (even though we don't seem to have any tests that test the behavior).